pyodide_build/out_of_tree/venv.py

```diff
@@ -192,8 +228,9 @@ def create_pip_script(venv_bin):
     ]
 
     for pip in other_pips:
-        pip.unlink()
-        pip.symlink_to(venv_bin / "pip")
+        if pip.exists():
+            pip.unlink()
```

Comment:
How about cleaning up all the files in the very first place of the function instead of calling `unlink` in several places?

Reply:
Makes sense – addressed in ec3ad527851c6cd66023b1e6201b615f7cb34e39